rules/linux/process_creation/proc_creation_lnx_iptables_flush_ufw.yml

```
@@ -0,0 +1,41 @@
+title: Flush Iptables Ufw Chain
```

```suggestion
title: Flush Iptables Ufw Logging
```
Seen to be the logging and not the filtering

---

It's using the flags that flush chains. The logging bits are just to tighten the rule